README.md

```diff
@@ -35,251 +13,97 @@ After you're finished please remove all the comments and instructions!
-```
---->
+- A computer running any operating system.
+- an internet connection
+- Have Git installed
+- A modern browser eg. Google Chrome
+- Knowledge of basic git commands
 
-### Run tests
+## Setup
 
-To run tests, run the following command:
+- link your computer to github
+- clone the repository
+- open the file in your code editor
+- set up Html and css linters for the project
 
-<!--
-Example command:
+## Install
 
-```sh
-  bin/rails test test/models/article_test.rb
-```
---->
+- install node_models and project dependencies
+- ensure there are no errors
 
-### Deployment
+## Usage
 
-You can deploy this project using:
+- run the project to ensure there are no linter errors
+- open index.html in your browser of choice
+- Enjoy!
 
-<!--
-Example:
+## Author
 
-```sh
+Samuel N. Kyere
 
-```
- -->
+GitHub: @admirerbrown
+Twitter: @brown_admirer
+LinkedIn: Samuel Ntow kyere
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- AUTHORS -->
-
-## 👥 Authors <a name="authors"></a>
-
-> Mention all of the collaborators of this project.
-
-👤 **Author1**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-👤 **Author2**
-
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- FUTURE FEATURES -->
-
-## 🔭 Future Features <a name="future-features"></a>
-
-> Describe 1 - 3 features you will add to the project.
-
-- [ ] **[new_feature_1]**
-- [ ] **[new_feature_2]**
-- [ ] **[new_feature_3]**
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- CONTRIBUTING -->
-
-## 🤝 Contributing <a name="contributing"></a>
+## Contributing
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [issues page](../../issues/).
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- SUPPORT -->
-
-## ⭐️ Show your support <a name="support"></a>
-
-> Write a message to encourage readers to support your project
-
-If you like this project...
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- ACKNOWLEDGEMENTS -->
-
-## 🙏 Acknowledgments <a name="acknowledgements"></a>
-
-> Give credit to everyone who inspired your codebase.
-
-I would like to thank...
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-<!-- FAQ (optional) -->
-
-## ❓ FAQ <a name="faq"></a>
-
-> Add at least 2 questions new developers would ask when they decide to use your project.
-
-- **[Question_1]**
-
-  - [Answer_1]
+## Acknowledgments
 
-- **[Question_2]**
+- Hat tip to Microverse for providing the resources that enabled me to do this work
+- Speacial credits to @ 2015 Creative Commons Korea for providing this amazing behance design.
 
```

Comment:
- [ ] It would be better if you add the link to the designer Behance page so others can get more information about the designer.

for instance 
```
Original design idea by [Cindy Shin in Behance](https://www.behance.net/adagio07).
```